Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.module.scss

```diff
@@ -57,6 +57,19 @@
   flex: 3;
 }
 
+.resourceWrapper {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  min-height: 50px;
+}
+
+.resourceLinkButton {
+  color: colors.$text-dark-gray-color;

```

Comment:
![image](https://user-images.githubusercontent.com/19411274/233354265-0c6e46c5-b22b-42e3-bfb4-58ee9cd0a0a5.png)
Dont see a reason why this button looks different than any other button in the app

Reply:
Reverted back to the default button color.